Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
> Why not one line? (I am no professional programmer, but I learn everyday)

We try to wrap as close as possible to 80 characters to keep the files easily readable and consistent. Doesn't always happen, but where we can it's nice to get it done.

> linkapps problematic?

Because they're symlinks to an often-hidden system location they don't show up in Spotlight or Alfred or similar things. The links also aren't automatically removed when the formula is, which is messy.